content/docs/news/vulnerability_bulletins/keptn-sec-2020-003/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Keptn-Vulnerability-2020-003
+description: RBAC cluster-admin role given to Keptn services by default
+weight: 98
+---
+
+Version 1.0
+
+## Revision Information 
+
+| Revision |    Updated   |     Reason     |
+|----------|:------------:|:--------------:|
+| 1.0      | December 22, 2021 | Initial Reason |
+
+## Affected
+
+Keptn jmeter-service in version 0.11.3 and older.
+
+## Description
+
+Keptn jmeter-service in version 0.11.3 and older includes JMeter with a version below v5.4.2 that is vulnerable to Log4j. With JMeter v5.4.2 the bug is fixed. [see](https://jmeter.apache.org/changes.html)
+
+## Severity
+
+*of Log4j:*
+
+  * **CVE-2021-44228** [details](https://nvd.nist.gov/vuln/detail/CVE-2021-44228)
+
+  * **CVE-2021-45046** [details](https://nvd.nist.gov/vuln/detail/CVE-2021-45046)
+
+## Recommendations
+
+The release of **jmeter-service** in version **0.11.4** contains the new JMeter version v5.4.2. 
+
+We recommend upgrading your Keptn control plane to 0.11.x as soon as possible allowing you to upgrade the **jmeter-service** to version **0.11.4**.
+
+Please follow this [upgrade path](https://keptn.sh/docs/0.11.x/operate/upgrade/#upgrade-from-keptn-0-11-x-to-keptn-0-11-4).
+
+## Workaround
+
+Stop using jmeter-service on your execution plane. 
```

Comment:
```suggestion
Stop using and disable jmeter-service on your execution plane. 
```

Reply:
This feels like a weird proposal for delivery pipelines. 

Ensure the service is not accessible from outside the execution plane?

Reply:
Thx, considered the feedback.